packages/dashboard-frontend/src/Layout/Header/Tools/about-modal.tsx

```diff
@@ -48,9 +49,17 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = (
     <>
       <TextContent>
         <TextList component='dl'>
+          {dashboardVersion && (
+            <>
+              <TextListItem component='dt'>Dashboard Version</TextListItem>
+              <TextListItem component='dd'>
+                <div className='co-select-to-copy'>{dashboardVersion}</div>
+              </TextListItem>
+            </>
+          )}
           {productVersion && (
             <>
-              <TextListItem component='dt'>Version</TextListItem>
+              <TextListItem component='dt'>Product Version</TextListItem>
```

Comment:
Maybe it's better to report it as `Server Version` since it's version of Che/CRW Server.
@nickboldt WDYT?

Reply:
Yes but in like 3 months "server" will be "devworkspace engine" ...

What's wrong with Product Version?

Reply:
Yes and no. We still will have Che Server for some duties. But its exact version may be not critical, so I'm OK with leaving Product as well.

About DevWorkspace operator version, we can try to report DWO CSV version, but in addition.